File: mcached/ocaml/read/mcached_read.ml
(* ------------------------------------------------------------------- *)
(*
	mcached_read.ml

					Feb/08/2011
*)
(* ------------------------------------------------------------------- *)
open String
open Memcache
open Json_type
open Json_type.Browse

open Mcached_manipulate

(* ------------------------------------------------------------------- *)
let _ = print_endline "*** 開始 ***"

let _ = Printf.printf "version: %s\n" Memcache.version

let cache = Memcache.open_connection "localhost" 11211

let keys = ["t1731"; "t1732"; "t1733";"t1734";"t1735";
	 "t1736"; "t1737";"t1738";"t1739"]

let _= out_proc cache keys

let _ =  Memcache.close_connection cache

let _ = print_endline "*** 終了 ***"
(* ------------------------------------------------------------------- *)

File: mcached/typescript/read/mcached_read.ts
// -------------------------------------------------------------
//	mcached_read.ts
//
//					Feb/11/2016
// -------------------------------------------------------------
declare function require(x: string): any;
declare var process:any;

var memcache = require('memcache')
var underscore = require('underscore')

var node_common = '/var/www/data_base/common/node_common'
var json_manipulate=require (node_common + '/json_manipulate')
// -------------------------------------------------------------
function proc01 (key)
{
	client.get(key, function(error,reply)
		{
		if (!error)
			{
			if (reply != null)
				{
				json_manipulate.json_display_proc (key,reply)
				}
			}
		})
}

// -------------------------------------------------------------
var onConnect = function()
{
	console.log("*** 開始 ***")
	var keys: string[] = ['t1731','t1732','t1733',
		't1734','t1735','t1736',
		't1737','t1738','t1739']

	for (var it in keys)
		{
		proc01 (keys[it])
		}

	client.close()
	console.log("*** 終了 ***")
}


// -------------------------------------------------------------
var client = new memcache.Client (11211,'localhost')
client.connect()
client.addHandler(onConnect)

// -------------------------------------------------------------

